okhttp/src/test/java/okhttp3/CallTest.java

```diff
@@ -939,20 +941,26 @@ private void postBodyRetransmittedAfterAuthorizationFail(String body) throws Exc
   }
 
   /** https://github.com/square/okhttp/issues/4761 */
-  @Test public void interceptorCallsProceedWithoutClosingPriorResponse() throws Exception {
+  @Test
+  public void interceptorCallsProceedAndClosesPriorResponse() throws Exception {
```

Comment:
why did the name change here? It doesn't close the prior response

Reply:
I've reverted name change, but I intended it to cover that the connection is forcibly closed in this case.